### PR TITLE
NMS-10582: snmp authentication error traps with Enhanced Linkd / bridge discovery

### PR DESCRIPTION
**Pull request by patrick-schweizer**
* JIra: https://issues.opennms.org/browse/NMS-10582

### CODE REVIEW

File: features/enlinkd/config/src/main/resources/xsds/enlinkd-configuration.xsd [hunk 1]
```diff
@@ -132,6 +132,14 @@
 					</documentation>
 				</annotation>
 			</attribute>
+			<attribute name="disable-bridge-vlan-discovery" type="boolean"
+					   use="optional" default="false">
+				<annotation>
+					<documentation>
+						Set to true to avoid errors as described in https://issues.opennms.org/browse/NMS-10582.
```
Comment (j-white) [inline features/enlinkd/config/src/main/resources/xsds/enlinkd-configuration.xsd]:
```suggestion
						Set to true to skip VLAN enumeration and scanning during bridge discovery.
```

File: opennms-base-assembly/src/main/filtered/etc/enlinkd-configuration.xml [hunk 2]
```diff
@@ -11,5 +11,5 @@
                      bridge_topology_interval="300000"
                      max_bft="100"
                      discovery-bridge-threads="1"
+                     disable-bridge-vlan-discovery="false"
```
Comment (j-white) [inline opennms-base-assembly/src/main/filtered/etc/enlinkd-configuration.xml]: We can omit changing this file.

Comment (patrick-schweizer) [reply opennms-base-assembly/src/main/filtered/etc/enlinkd-configuration.xml]: fixed.

File: opennms-base-assembly/src/main/filtered/etc/enlinkd-configuration.xml [hunk 3]
```diff
@@ -12,4 +12,3 @@
                      max_bft="100"
                      discovery-bridge-threads="1"
                      />
-
```
Comment (j-white) [inline opennms-base-assembly/src/main/filtered/etc/enlinkd-configuration.xml]: Unnecessary change.

Comment (patrick-schweizer) [reply opennms-base-assembly/src/main/filtered/etc/enlinkd-configuration.xml]: fixed.
